app/build.gradle

```diff
@@ -2,11 +2,13 @@ apply plugin: 'com.android.application'
 apply from: "$rootDir/config/feature_dependencies.gradle"
 apply plugin: 'kotlin-parcelize'
 apply from: 'https://raw.githubusercontent.com/JakeWharton/SdkSearch/master/gradle/projectDependencyGraph.gradle'
+apply plugin: 'dagger.hilt.android.plugin'
 
 
 dependencies {
     implementation project(':data')
     implementation project(':usecases')
+    implementation 'com.google.dagger:hilt-android:2.38.1'
```

Comment:
Since you are adding the dependencies on a buildSrc module, maybe it is a good idea to add this one there too. 😊

Reply:
Ooops... missed that one!